bioconda_utils/utils.py

```diff
@@ -846,6 +845,16 @@ def check_recipe_skippable(recipe, channel_packages, force=False):
         ),
         Counter()
     )
+    if num_existing_pkg_builds == Counter():
+        # No packages with same version + build num in channels: no need to skip
+        return False
```

Comment:
If
```python
num_existing_pkg_builds == num_new_pkg_builds == Counter()
```
this previously caused an additional
> ... the same number of builds are in channel(s) ...

info message to be displayed. But since
```python
num_new_pkg_builds == Counter()
```
means there is nothing to do anyway,
> Nothing to be done for recipe ...

is output in any case so that the former message is just noise.
ref: https://github.com/bioconda/bioconda-recipes/pull/9686#issuecomment-401799336